docs/coverage-reporter/uploading-coverage-in-advanced-scenarios.md

```diff
@@ -10,8 +10,11 @@ The following sections include instructions on how to use the Codacy Coverage Re
 
 If your test suite is split in different modules or runs in parallel, you must upload multiple coverage reports for the same language either at once or in sequence.
 
+!!! note
+    A line of coverable code can only be "covered" or "uncovered". If multiple reports mark a line as covered, or if a single report marks a line as covered multiple times, it always counts as a single covered line when calculating coverage.
```

Comment:
Some suggestions:

- Make it more explicit that it's **from the standpoint of Codacy** that the coverage status of a line of code is binary, either covered or not covered. For example: "On Codacy, a coverable line of code can only be covered or not covered," and "Codacy always counts as a single covered line when calculating coverage."

- Use the same nomenclature that we're already using in pages such as [Why does Codacy show unexpected coverage changes?](https://docs.codacy.com/faq/code-analysis/why-does-codacy-show-unexpected-coverage-changes/), such as:

    - "line of coverable code" :arrow_right: "coverable line [of code]"
    - "uncovered" :arrow_right: "not covered"

- Finally, experiment with including this information in a straight paragraph following the existing intro. Maybe we don't need an extra admonition for this clarification since it's so closely tied to the topic of sending multiple reports that can potentially "overlap"?